### PR TITLE
Document various log codes

### PR DESCRIPTION
**Pull request by nkolev92**
Fixes https://github.com/NuGet/docs.microsoft.com-nuget/issues/2026
Fixes https://github.com/NuGet/docs.microsoft.com-nuget/issues/2549
Fixes https://github.com/NuGet/docs.microsoft.com-nuget/issues/2899
Fixes https://github.com/NuGet/docs.microsoft.com-nuget/issues/2524
Fixes https://github.com/NuGet/docs.microsoft.com-nuget/issues/2553
Fixes https://github.com/NuGet/docs.microsoft.com-nuget/issues/2558

Documenting: NU1008, NU1009, NU1010, NU1011, NU1109, NU1402, NU1403, NU5042, NU5049

### CODE REVIEW

**Review by erdembayar — APPROVED** [review]

**Review by zivkan — APPROVED** [review]

File: docs/reference/errors-and-warnings/NU1008.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+Example:
+
+```xml
+<!-- In the project file. -->
+<PackageReference Include="PackageId" />
+```
+
+```xml
+<!-- In the project file -->
```
Comment (zivkan) [inline docs/reference/errors-and-warnings/NU1008.md:40]:
```suggestion
<!-- In the Directory.Packages.props file -->

```

File: docs/reference/errors-and-warnings/NU1008.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+<pre>Projects that use central package version management should not define the version on the PackageReference items but on the PackageVersion items: PackageId.</pre>
+
+### Issue
+
+When using central package management, versions must be defined on the PackageVersion item.
+
+### Solution
+
+- Remove the version from the PackageId PackageReference.
+- You may need to add or update the PackageVersion item for PackageId.
```
[the remark this answers is not in the document]
Comment (nkolev92) [reply docs/reference/errors-and-warnings/NU1008.md]: Addressed.

File: docs/reference/errors-and-warnings/NU1008.md (hunk 1 above)
Comment (nkolev92) [reply docs/reference/errors-and-warnings/NU1008.md]: I saw it after I pushed, so I'm choosing not to give you credit for this.

Comment (zivkan) [reply docs/reference/errors-and-warnings/NU1008.md]: Do I at least get credit for reviewing your change within 120 seconds of when you pushed it?

File: docs/reference/errors-and-warnings/NU1010.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+<pre>The PackageReference items PackageId do not have corresponding PackageVersion.</pre>
+
+### Issue
+
+The PackageReference PackageId is missing a PackageVersion item.
+
+### Solution
+
+Add a PackageVersion item for PackageId.
```
[the remark this answers is not in the document]
Comment (nkolev92) [reply docs/reference/errors-and-warnings/NU1010.md]: Fixed.

File: docs/reference/errors-and-warnings/NU1403.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+### Issue
+The content of the package in the global packages folder is different from the value in the lock file.
+The package on the source may have changed, or a multiple sources may have different instances of the same package id and version.
+
+### Solution
+
+* If you believe that the version you have installed locally is the up to date, correct one, run restore with the [force evaluate](../../consume-packages/Package-References-in-Project-Files.md#lock-file-extensibility) option to update the lock file to the local version of the package.
+
+* [Inspect the package in the global packages folder](../../concepts/troubleshooting-installed-packages.md#nupkgmetadata-file-in-global-packages-folder) and determine the source from which it was installed. If you own the source in quesiton, you may need to ensure the correct version of the package is available. Once you have figured out which package version you want, run restore with [force evaluate](../../consume-packages/Package-References-in-Project-Files.md#lock-file-extensibility).
```
[the remark this answers is not in the document]
Comment (nkolev92) [reply docs/reference/errors-and-warnings/NU1403.md:24]: Great point. Added a simple thing. Not sure it's the most actionable one, but I don't really know what more detail to provide there.

File: docs/reference/errors-and-warnings/NU1011.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+# NuGet Error NU1011
+
+<pre>Centrally defined floating package versions are not allowed.</pre>
+
+### Issue
+
+PackageVersion items cannot contain floating versions.
+### Solution
+
+Change the floating version to a standard range.
```
[the remark this answers is not in the document]
Comment (nkolev92) [reply docs/reference/errors-and-warnings/NU1011.md]: Great point added.

**Comment by nkolev92** [comment]
@zivkan Addressed all your feedback, in case you want to look again, but I'm pretty sure I got it right :D

**Review by zivkan — APPROVED** [review]
lgtm

**Pull request merged**